### ISSUE TITLE
go to symbol in the terminal's accessible view doesn't contain all commands after the first invocation

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#192812]
1. with screen reader mode enabled, run some commands in the terminal
2. `ctrl/cmd+up arrow` to open the accessible view
3. `ctrl/cmd+shift+o` to go to symbol
4. ✅  the commands are there
5. `Escape` then `ctrl/cmd+shift+o` again
6. 🐛 some commands are missing (only the most recent one is there)
